plugins/obs-ffmpeg/CMakeLists.txt

```diff
@@ -66,6 +73,8 @@ target_link_libraries(
           $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:Libva::va>
           $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:Libva::drm>
           $<$<PLATFORM_ID:Linux,FreeBSD,OpenBSD>:Libpci::pci>
+          $<$<AND:$<BOOL:${ENABLE_NATIVE_NVENC}>,$<PLATFORM_ID:Linux>>:OBS::obs-nvenc-version>
+          $<$<AND:$<BOOL:${ENABLE_NATIVE_NVENC}>,$<PLATFORM_ID:Linux>>:FFnvcodec::FFnvcodec>
```

Comment:
On Linux, this needs to also link to Glad (`OBS::glad` if I'm not misreading things) for the CUDA/OpenGL interop.